runtime/oti/j9javaaccessflags.h

```diff
@@ -44,6 +44,7 @@
 #define J9AccClassDying 0x8000000
 #define J9AccClassFinalizeNeeded 0x40000000
 #define J9AccClassGCSpecial 0x800000
+#define J9GCSpecialHotField 0x800000 
```

Comment:
Is this a class flag?  It seems to have the same value as the flag above (`J9AccClassGCSpecial`) but a different name

Reply:
There's also a comment in createramclass.cpp that will need to be updated to include any new flag bits

Reply:
This is not a class flag like J9AccClassGCSpecial. It is the bit I use in the hotFieldOffset field I added in J9Class that lets me know if it the hotFieldOffset is the hotFieldOffset of one of my special classes I define in ObjectModel.cpp (String, Hashmap, Treemap etc.)

Reply:
This should declared slightly off these flags to avoid confusion. It has nothing to do with class flags.

Also I suggest renaming - instead of Special it could be Predefined?